src/zenml/integrations/kubernetes/orchestrators/yaml/jupyter.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
```

Comment:
As discussed, we should probably make these deployment yamls a bit friendlier and approachble in the src code, probably with their own README in the src code

Reply:
Rewrote them in source code using kubernetes API client.